.rubocop.yml

```diff
@@ -123,3 +123,6 @@ Style/TrivialAccessors:
 
 Style/WordArray:
   Enabled: false
+
+Style/ColonMethodCall:
+  Enabled: false
```

Comment:
What I mean is why don't you use `Unicode.downcase` instead of `Unicode::downcase`? Then you wouldn't need this rule here.